bin/_util.js

```diff
@@ -171,25 +171,17 @@ module.exports = {
 
     /**
      * Retrieve the defined variable collections from the specific figma file data
-     * @see https://www.figma.com/plugin-docs/api/VariableCollection/
+     * @see https://www.figma.com/developers/api#variables
      *
      * @returns VariableCollection[]
      */
     getVariableCollections() {
-      // TODO-AH: remove the redundant rows in here by making this a unique set
-      return Object.values(this._jsonData.variableCollections).map(
```

Comment:
no longer picking some data from `VariableCollecton` to return. Instead, give the caller the full-fat data